lib/fake-xhr/index.js

```diff
@@ -332,12 +332,17 @@ FakeXMLHttpRequest.parseXML = function parseXML(text) {
         try {
             if (typeof DOMParser !== "undefined") {
                 var parser = new DOMParser();
-                return parser.parseFromString(text, "text/xml");
+                var parsererrorNS = parser.parseFromString("INVALID", "text/xml")
+                    .getElementsByTagName("parsererror")[0].namespaceURI;
+                var result = parser.parseFromString(text, "text/xml");
+                return result.getElementsByTagNameNS(parsererrorNS, "parsererror").length
+                    ? null : result;
             }
             var xmlDoc = new window.ActiveXObject("Microsoft.XMLDOM");
             xmlDoc.async = "false";
             xmlDoc.loadXML(text);
-            return xmlDoc;
+            return xmlDoc.parseError.errorCode !== 0
```

Comment:
This is what MSDN says should happen, but I have no way of testing if that is actually correct unfortunately.

Reply:
Just got my hands on IE11 and it seems like this actually works as advertised. Alternatively is looks like the `loadXML` call returns `true` or `false` depending on if parsing worked, but since the above is what's documented, I'd stick to it.